### DISCUSSION TITLE
Renamed connection in circuitboxes won't save upon copypasting it

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#15269] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Renamed connections wont be saved if you copy paste the circuit box or save it as an assembly and paste it. 

### Reproduction steps

1. Rename circuit box connections
2. Copy and paste it

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by CRedinger** [comment]
It's a shame this awesome feature is bugged.

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in our private repo, fix coming up in the next unstable build and the upcoming release.
